lib/runner.js

```diff
@@ -1,8 +1,5 @@
 'use strict';
 
-/**
- * @module Runner
- */
```

Comment:
Why was this removed?

Reply:
because it conflicts with the class name.  we're using `@module` wrong, basically.

when referring to e.g. `Runner#run`, we want to write `Runner#run`; when we use modules this way we have to do `module:Runner~Runner#run` or some such nonsense.

since all/most of these modules are exported via `Mocha` itself (e.g. `Mocha.Runner`), and the default export is the class (`Runner`) we don't need to use `@module` _in these cases_.

the docstrings are screwy anyway, but since I needed to modify API docs I thought I'd try to make it consistently output for constant-defining classes, which it is now.

Reply:
I think I also removed `@module Suite`.